server/ai_mediaserver.go

```diff
@@ -440,14 +440,18 @@ func (ls *LivepeerServer) StartLiveVideo() http.Handler {
 		}
 		// If auth webhook is set and returns an output URL, this will be replaced
 		outputURL := qp.Get("rtmpOutput")
-
-		mediaMTXOutputURL := fmt.Sprintf("rtmp://%s/aiWebrtc/%s-out", remoteHost, streamName)
 		if outputURL == "" {
 			// re-publish to ourselves for now
 			// Not sure if we want this to be permanent
 			outputURL = fmt.Sprintf("rtmp://%s/%s-out", remoteHost, streamName)
 		}
 
+		internalOutputHost := os.Getenv("LIVE_AI_PLAYBACK_HOST") // TODO proper cli arg
+		if internalOutputHost == "" {
+			internalOutputHost = remoteHost
+		}
+		mediaMTXOutputURL := fmt.Sprintf("rtmp://%s/aiWebrtc/%s-out", internalOutputHost, streamName)
```

Comment:
Encode the `aiWebrtc` in `LIVE_AI_PLAYBACK_HOST` so we are not hard-coding paths which are specific to our infra setup - that is what we do with the new WHIP server.

```suggestion
		if internalOutputHost == "" {
			internalOutputHost = fmt.Sprintf("rtmp://%s/aiWebrtc/", remoteHost)
		}
		mediaMTXOutputURL := internalOutputHost + streamName + "-out"

```

Reply:
Thanks Josh yep I blindly thought input was already working, I've made some changes as per your suggestion